### ISSUE TITLE
Style past events differently

### ISSUE THREAD

**Issue by Restuta** [Restuta/rcn.io#57]
Events that are in the "past" should have different style, few ideas:
* they should be faded so visually it's immediately obvious what is "past"
* event should have an obvious link to results (in future current racer placement would be great)

**Comment by Restuta** [comment]
![](http://media2.giphy.com/media/11WyKmlnDY3mQ8/giphy.gif)

**Comment by Restuta** [comment]
something like this:
<img width="908" alt="2016-02-18_1944" src="https://cloud.githubusercontent.com/assets/377261/13166035/2aeba928-d678-11e5-9347-900ad93c9747.png">

**Comment by Me4ta** [comment]
Cancelled races should be faded too, I think.

**Comment by Restuta** [comment]
@Me4ta agree, not only faded, but a clear indication _why_ they are faded, like a word "CANCELLED"
